.github/workflows/dest_test.yml

```diff
@@ -0,0 +1,32 @@
+name: Destination Plugin Test Workflow
+
+on:
+  pull_request:
+    paths:
+      - "plugins/destination/test/**"
```

Comment:
Regardless where we land, the `paths` is missing `.github/workflows/dest_test.yml`, otherwise changes to the workflow file won't trigger it (for example in a PR)